File: Android/Java/HealthFitness/app/src/main/java/com/example/healthfitness/MainActivity.java
package com.example.healthfitness;

import androidx.appcompat.app.AppCompatActivity;
import androidx.cardview.widget.CardView;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {
    CardView cardView;
    TextView textView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        cardView = findViewById(R.id.cardview);
        textView = findViewById(R.id.textview);

        new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                Intent intent = new Intent(MainActivity.this, SecPage.class);
                startActivity(intent);
                finish(); // Close the splash activity to prevent going back to it
            }
        }, 3000);
    }
}

File: Android/Java/RecipeApp/app/src/main/java/com/example/recipeapp/MainActivity.kt
package com.example.recipeapp

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import java.time.Instant

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        Handler(Looper.getMainLooper())
            .postDelayed({
                startActivity(Intent(this,HomeActivity::class.java))
                finish()
            },2000
            )

    }
}